pkg/clusteragent/autoscaling/workload/controller_horizontal.go

```diff
@@ -204,6 +204,30 @@ func (hr *horizontalController) computeScaleAction(
 		return nil, 0, errors.New(reason)
 	}
 
+	var evalAfter time.Duration
```

Comment:
Is there a reason why it's early in the flow? I would expect stabilization to be after the `outsideBoundaries` check and after flooring `targetDesiredReplicas` between `min and max`?